src/helpers/loadFont.ts

```diff
@@ -5,7 +5,7 @@ export default (url: string) => new Promise((resolve) => {
   }
 
   if (document.head.querySelector(`link[href="${url}"]`)) {
-    resolve();
+    resolve({});
```

Comment:
why?

Reply:
`resolve` expects a `value: unknown` passed unless we declare `Promise<void>` but if we do so then TS will complain about  `link.onLoad = resolve`:
```
Type '(value: void | PromiseLike<void>) => void' is not assignable to type '(this: GlobalEventHandlers, ev: Event) => any'.
  Types of parameters 'value' and 'ev' are incompatible.
    Type 'Event' is not assignable to type 'void | PromiseLike<void>'.
      Property 'then' is missing in type 'Event' but required in type 'PromiseLike<void>'.ts(2322)
 ```
 
 Passing anything to `resolve(...)` helps, but I see we can also change mentioned line to `link.onload = () => resolve();`